cluster-service/Makefile

```diff
@@ -26,6 +26,7 @@ deploy:
 	  -p CS_SERVICE_PRINCIPAL_CREDS_BASE64=$${CS_SERVICE_PRINCIPAL_CREDS_BASE64} \
 	  -p IMAGE_REGISTRY=${ARO_HCP_IMAGE_ACR}.azurecr.io \
 	  -p IMAGE_REPOSITORY=app-sre/uhc-clusters-service \
+	  -p AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID=$${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID} \
```

Comment:
Need to set the value for `$${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID}` before we use it in oc process. Maybe @janboll can help here with the az command to retrieve it?

Reply:
added the retrieval